src/main/resources/db/migration/V15__add_collection_coverage_table.sql

```diff
@@ -0,0 +1,9 @@
+-- create collection_coverage table with foreign key constraint
+CREATE TABLE collection_coverage (
```

Comment:
- You can add GRANT statements for the table. See migration no. 11 as a reference.
- For the foreign key, you can add `ON DELETE CASCADE` (which makes removing data from the DB a little easier). See migration no. 12 as a reference.